cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -983,7 +983,7 @@
 
     <property>
         <name>router.server.address</name>
-        <value>localhost</value>
+        <value>0.0.0.0</value>
```

Comment:
This should be the address used to connect to the router. I think the default of localhost is fine, here. Setting it to `0.0.0.0` means anything that uses this file, like the UI, won't know what to connect to.